### PR TITLE
MYST-187 remove client create session

### PR DESCRIPTION
**Pull request by donce**
This should be merged only after https://github.com/MysteriumNetwork/node/pull/69

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -76,44 +75,6 @@ def proposals():
...
-@validate_json
-def client_create_session():
-    payload = request.get_json(force=True)
-    node_key = payload.get('node_key', '')
-
-    node = Node.query.get(node_key)
-    if not node:
-        return jsonify(error='node key not found'), 400
-
-    if node.get_status() != 'active':
```
Comment (Waldz) [inline app.py]: Are You listing just `active` proposals in `v1/proposals`?

Comment (donce) [reply app.py]: Yes - this logic is in the model itself.

File: app.py [hunk 2]
```diff
@@ -76,44 +75,6 @@ def proposals():
...
-        return jsonify(error='cannot deserialize service proposal'), 400
-
-    service_proposal = obj.get('service_proposal', None)
-    if service_proposal is None:
-        return jsonify(error='service_proposal was not found'), 400
-
-    session_key = payload.get('session_key') or helpers.generate_random_string()
-    session = Session(session_key)
-    session.node_key = node_key
-    session.client_updated_at = datetime.utcnow()
```
Comment (Waldz) [inline app.py]: We need new sessionSaving endpoint, with same features

Comment (donce) [reply app.py]: We decided to save session in stats endpoints, not on a separate endpoint.
On stats endpoint, new session will be created with `session_id` if it does not exist already.

Comment (Waldz) [reply app.py]: Just make a note and dont forget fields:
- `client_updated_at`
- `client_ip`

Comment (donce) [reply app.py]: Added a note here: https://mysteriumnetwork.atlassian.net/browse/MYST-142

**Pull request merged**